robottelo/config/base.py

```diff
@@ -306,13 +306,15 @@ class DockerSettings(FeatureSettings):
     """Docker settings definitions."""
     def __init__(self, *args, **kwargs):
         super(DockerSettings, self).__init__(*args, **kwargs)
-        self.unix_socket = None
```

Comment:
I'd keep this bool, to state you have internal docker setup and ready to be tested.

To have something to decorate ```DockerUnixSocketContainerTestCase```
```@skip_if(settings.docker.unix_socket)```